GitUI/SpellChecker/EditNetSpell.cs

```diff
@@ -750,7 +750,22 @@ private void PasteTextFromClipboard()
 
         private void TextBox_KeyPress(object sender, KeyPressEventArgs e)
         {
-            _disableAutoCompleteTriggerOnTextUpdate = e.KeyChar.IsSeparator();
+            bool isSeparator = e.KeyChar.IsSeparator();
+            _disableAutoCompleteTriggerOnTextUpdate = isSeparator;
+
+            if (e.KeyChar == '\b')
```

Comment:
What's `\b`?

Reply:
https://learn.microsoft.com/en-us/cpp/c-language/escape-sequences?view=msvc-170

Reply:
It's the key you receive when you delete a character that's why I put a comment just after.

Reply:
from SimplePrompt: `if (e.KeyChar == (char)Keys.Escape)`